### PR TITLE
Pair programming contact form

### PR DESCRIPTION
**Pull request by darikmohammed**
# Created a Contact form 
> We were able to do pair programming through this branch. 
## Completed Tasks
- Created a form to submit a contact message.
- Integrate the form with the Formspree service.

https://user-images.githubusercontent.com/56404835/165370058-0e1fb45a-317f-4233-9480-cf6d9426e2f8.mov

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -266,3 +266,47 @@ p {
...
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+}
+
+.flex-item-2 {
+  align-self: stretch;
+}
```
Comment (danmainah) [inline style.css:288]: - [ ] Kindly use flexbox to position your elements, as instructed in the project requirements.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @darikmohammed , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️

- [ ] Kindly ensure the design matches the Figma design, see the comparison below for the small hitch to fix by adding background image on the side.

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image| image



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -190,5 +189,73 @@ <h3 class="card-discription">Skills</h3>
...
+                maxlength="30"
+                placeholder="Full Name"
+              />
+            </li>
+            <li>
+              <input
+                type="email"
+                name="email"
+                required
+                placeholder="Email Address"
```
Comment (gtekle) [inline index.html:219]: - [ Optional ] In my opinion, you could have achieved the same thing without using `<ul> <li>` in here. Besides you code could have been leaner and more readable. e.g. 

```
<form action>
  <input name="fullname"/>
  <input name="email"/>
  <textarea  name="message">
  <button type="submit">
</form>
```

**Review by gtekle — APPROVED** [review]
Hi **Team**, 

## APPROVED :clap: :clap: :clap: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
Got Talent-congrats


## Highlights
:heavy_check_mark: Meaningful commit messages
:heavy_check_mark: Nice UI

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**